product_analytic_sale/models/sale_order_line.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    def _compute_analytic_distribution(self):
+        """
+        Get analytic distribution from product expense analytic account
```

Comment:
Why do we mention `expense` here?

Reply:
@rousseldenis did you get time to go ahead with the comment of Stéphane ?